web/src/modules/user/profile/types.ts

```diff
@@ -47,6 +47,7 @@ export interface BitzlatoUser {
     verified: boolean;
     timezone?: string | null | undefined;
     safe_mode_enabled: boolean;
+    self_frozen?: boolean;
```

Comment:
```suggestion
    self_frozen: boolean;
```
Это свойство обязательное

Reply:
ок!